### ISSUE TITLE
Definition of 3D is to low

### ISSUE THREAD

**Issue by ThorvaldAagaard** [EdwardPiwowar/BBA#146]
380594878F69EEDA15BDDAC698B9
![image](https://github.com/EdwardPiwowar/BBA/assets/10560652/4709bf52-08a4-471e-8ff6-7ffa09683206)
3D is more like 6-9 - and 2S is a limit+ raise 10+ hcp

**Comment by ThorvaldAagaard** [comment]
0D72EEC8C14F14372B5A8B0924F2
![image](https://github.com/EdwardPiwowar/BBA/assets/10560652/e7122ccc-1749-41e2-9d21-e14e4ad06e34)
But here is North not bidding 3D with a better hand. Is it the X that cause problems?

**Comment by EdwardPiwowar** [comment]
Imo, 3D is blocking, 2S is a limit+ raise 8+ HCP, 10+ is is very unlikely bacause less than 10 HCO left.
Second hand: there is no reason to block.

**Comment by ThorvaldAagaard** [comment]
I do not agree. 3D is just 3D. A descriptive bid giving partner the information needed to make the right decision. The idea of passing on a hand with support and then support on the next round is not good bridge. Using limit raise with 8+ is also overbidding - limit raise should as default promise that we have at least 50% of the HCP's
